pkg/policy/distillery.go

```diff
@@ -153,6 +153,34 @@ func (cache *PolicyCache) UpdatePolicy(identity *identityPkg.Identity) error {
 	return err
 }
 
+// GetAuthType returns the AuthType required by the policy between the localID and remoteID, or
+// AuthTypeDisabled if none.
+//
+// Note that the first found AuthType is returned. This is correct as the policy should only
```

Comment:
technically, it's possible to have multiple auth types for different protocols & ports between the same identities.

so i wonder whether it would be ok to just return whether auth is enabled between the two identities?